projects/igniteui-angular/src/lib/grids/grid/grid-row.component.html

```diff
@@ -107,7 +107,6 @@
         [intRow]="this"
         [lastPinned]="col.columnLayoutChild ? null : col.isLastPinned"
         [firstPinned]="col.columnLayoutChild ? null : col.isFirstPinned"
-        [style.min-height.px]="cellHeight"
```

Comment:
Is min-height no longer needed?

Reply:
Yes, I changed the way the row gets its min-height. I have included it in the theme and it works with css now: https://github.com/IgniteUI/igniteui-angular/pull/14163/commits/8be56af333b6fb0a731190b13422f069cf857b56#diff-29930daadb3a8deb436ab411208d6c7f5008e2239dd5ecb3e6f3669b36d02e22R1434 . I have also removed the defaultRowHeight from the grid.